tests/test_import_source.py

```diff
@@ -115,3 +128,21 @@ def test_import_various_field_types(tmp_path, postgis_db, cli_runner):
         "reel": {"dataType": "float", "size": 64},
         "tumeric": {"dataType": "float", "size": 64},
     }
+
+
+def test_list_postgres_tables(postgis_db, postgres_table_with_types, cli_runner):
+    # Using postgres here because it has the best type preservation
```

Comment:
"# Using postgres here because it has the best type preservation"
That's not why you're using postgres here

Reply:
truth